### ISSUE TITLE
Redesign default layout

### ISSUE THREAD

**Issue by ulfaslak** [ulfaslak/network_styling_with_d3#22]
The old teal nodes are starting to hurt my eyes slightly. @benmaier do you have any suggestions for a new default layout? I'm thinking something black and simple along the lines of:

<img width="1066" alt="Screen Shot 2019-05-17 at 14 26 59" src="https://user-images.githubusercontent.com/6390218/57931517-eb639b80-78b8-11e9-8759-cc898569bee8.png">

**Comment by benmaier** [comment]
well since the default colors now represent the flag of Nazi Germany I'm heavily against it, but I'll be giving some more constructive feedback soon :)

<img width="131" alt="Screen Shot 2019-05-20 at 10 36 57" src="https://user-images.githubusercontent.com/10728380/58007745-382eb880-7aeb-11e9-8acf-03a6e4c876af.png">

**Comment by ulfaslak** [comment]
I need more sleep

**Comment by benmaier** [comment]
should we redesign the default options such that `'zoom': 1` is the default zoom? would make more sense tbh.

**Comment by ulfaslak** [comment]
Yes. I agree. Which means also setting the initial physics params slightly different.

**Comment by benmaier** [comment]
?

![Screen Shot 2019-05-23 at 14 59 34](https://user-images.githubusercontent.com/10728380/58254623-6cb0a780-7d6b-11e9-94bb-728af46b57e2.png)
